x-pack/legacy/plugins/apm/public/components/shared/charts/CustomPlot/StaticPlot.js

```diff
@@ -122,7 +143,19 @@ class StaticPlot extends PureComponent {
             })}
           />
         ) : (
-          this.getVisSeries(series, plotValues)
+          [
+            <HorizontalGridLines key="grid-lines" tickValues={yTickValues} />,
+            <YAxis
+              key="y-axis"
+              tickSize={0}
+              tickValues={yTickValues}
+              tickFormat={tickFormatY}
+              style={{
+                line: { stroke: 'none', fill: 'none' }
+              }}
+            />,
+            this.getVisSeries(series, plotValues)
+          ]
```

Comment:
This will remove the y-axis and the horizontal grid lines when there is no data. Probably not a big issue but why? 
Isn't it nice to keep these thing around so the chart doesn't "jump"?
<img width="1214" alt="Screenshot 2019-07-05 at 14 13 59" src="https://user-images.githubusercontent.com/209966/60721778-4ddb2e80-9f2f-11e9-8d9a-38e950b3adcc.png">

Reply:
agreed, that makes the charts behave differently, at least the transactions duration and requests per minute ones. 
<img width="1632" alt="Screenshot 2019-07-05 at 14 28 07" src="https://user-images.githubusercontent.com/4104278/60722432-2ab17e80-9f31-11e9-909f-bd26cd47e6fc.png">